src/user-guide/writing-workflows/external-triggers.rst

```diff
@@ -220,6 +220,47 @@ To see this, take a look at the job script for one of the downstream tasks:
    configuration.
 
 
+.. _Sequential Trigger Spawning:
+
+Sequential Trigger Spawning
+---------------------------
+
+Parent-less tasks (having no graph dependencies on other tasks) naturally spawn
+out to the runahead limit. The consequence for workflows with large runahead
+limits and/or task count may not only be UI clutter but also needless trigger
+checking. An example of this would be ``wall_clock``, where there's clearly
+no point in checking the future if the present trigger is unsatisfied.
```

Comment:
Spell check didn't like "Parentless" 

> If their xtriggers will only be satisfied in order anyway

Well, it may cause clutter even if satisfied out of order.. It's really up to the workflow designer if they want to constrain it unnaturally too.

Reply:
True.  In that case, how about:
> This may cause UI clutter, and unnecessary (or even pointless?) xtrigger checking if the xtriggers would only be satisfied in order.